miprometheus/workers/tester.py

```diff
@@ -356,9 +470,31 @@ def main():
     """
     tester = Tester()
     # parse args, load configuration and create all required objects.
-    tester.setup_experiment()
-    # GO!
-    tester.run_experiment()
+    tester.setup_global_experiment()
+
+    if tester.check_multi_tests():
```

Comment:
Ok, so this is how you solved it, i.e. outside of tester...

Reply:
Yes, had to check it outside of `setup_experiment` as I did not want to introduce too much overhead. In the case where the user does not indicate `multi_tests` (i.e. in most current config files), then the overhead is just the run time of `try -> except`, which should be negligible.

Reply:
sure, makes sense to me 👍